### ISSUE TITLE
[cards] Use image alt text and not article title

### ISSUE THREAD

**Issue by kptdobe** [adobe/business-website#100]
In the SEO analysis report that Matt shared for theblog, there is a requirement that on the homepage, images alt text should be the alt text describing the image and the article title.
Since the cards are rendered via the index, this means we must index the hero image alt text too and display it from the index. We would still fallback to the article title if no alt text as been defined on the image.
